Packs/IntegrationsAndIncidentsHealthCheck/Playbooks/playbook-JOB_-_Integrations_and_Playbooks_Health_Check.yml

```diff
@@ -1220,6 +973,18 @@ inputs:
   description: This input determines if the investigation should close automatically
     after the re-run of the scripts.
   playbookInputQuery:
+- key: playbookExclusion
+  value: {}
+  required: false
+  description: Comma Separated list of failed tasks to exclude from restart based
```

Comment:
Comma Separated list of failed tasks or playbooks? This sentence is a bit confusing for me